src/server/session.ts

```diff
@@ -1068,6 +1070,12 @@ export class Session<TMessage = string> implements EventSender {
     }
 
     private sendRequestCompletedEvent(requestId: number): void {
+        if (this.requestPerformanceData.has(requestId)) {

```

Comment:
I dont think you need request id in the map at all.. There cannot be another active "err" request before cancelling this one so you should be able to just use this as is just like `performanceData` field on session. 
We should just add the data in that one instead of creating new field and sending it across. so we are also sending data about update graph etc. #58845

Reply:
That makes sense. I wasn't sure if it was possible or not to end up having more than one error computation at the same time, because of it being async and cancellation, but that makes sense. I'll review #58845 and make my changes on top of that.